soc/arm/st_stm32/common/Kconfig.defconfig.series

```diff
@@ -9,8 +9,18 @@ if SOC_FAMILY_STM32
 
 config CORTEX_M_SYSTICK
 	bool
-	default n if STM32_LPTIM_TIMER
-	default y if !STM32_LPTIM_TIMER
+	default y
+	depends on !STM32_LPTIM_TIMER
+
+# set the tick per sec as a divider of the LPTIM clock source
+config SYS_CLOCK_TICKS_PER_SEC
+	default 4000
+	depends on STM32_LPTIM_TIMER && STM32_LPTIM_CLOCK_LSI
+
+# set the tick per sec as a divider of the LPTIM clock source
+config SYS_CLOCK_TICKS_PER_SEC
+	default 4096
+	depends on STM32_LPTIM_TIMER && STM32_LPTIM_CLOCK_LSE
```

Comment:
Like @aurel32 said (https://github.com/zephyrproject-rtos/zephyr/pull/25126#issuecomment-627639520), it could be interesting to have 32768 here instead of 4096

Reply:
Yes, but this value of 32768 for TICKS_PER_SEC is too high and cannot pass the kernel/tests timers_api

Reply:
testing kernel/tests on nucleo_l4r5zi with
CONFIG_STM32_LPTIM_TIMER=y1
CONFIG_SYS_CLOCK_TICKS_PER_SEC=32768